openedx/features/student_certificates/helpers.py

```diff
@@ -170,47 +164,40 @@ def get_philu_certificate_social_context(course, certificate):
     return social_sharing_urls
 
 
-def get_image_and_size_from_url(url):
+def get_pdf_data_by_certificate_uuid(uuid):
```

Comment:
Do we think there should be a unit test to separately test this helper? Or is testing the view good enough?

Reply:
Actually majorly this method is getting image using `requests.get` and then it convert the response content into bytes and then to pdf using `img2pdf.convert`. Both are the third party libraries, To write unit test we need to mock these things. That's why I didn't write unit test for this helper method